hackclub.com.yaml

```
@@ -2839,6 +2839,16 @@ scrapyard:
   - ttl: 60
     type: ALIAS
     value: cname.vercel-dns.com.
+  scriptingnest:
+  - ttl: 600
+    type: TXT
+    value: domain-verification=future-x
+  - ttl: 600
+    type: A
+    value: 37.27.51.34
+  - ttl: 600
+    type: AAA
```

Did you mean AAAA perhaps? :3
```suggestion
    type: AAAA
```